checkov/terraform_json/runner.py

```diff
@@ -177,7 +178,7 @@ def add_graph_check_results(self, report: Report, runner_filter: RunnerFilter) -
                 record.set_guideline(guideline=check.guideline)
                 report.add_record(record=record)
 
-    def run_block(
+    def run_block(  # type:ignore[override]  # would probably need to make 'TerraformRunner' generic
```

Comment:
why is this ignore needed? which type is not ok here?

Reply:
it is related to `definition_context` and `full_file_path`, which expect `TFDefinitionKey` instead of `str` in the parent.